pkg/config/config.go

```diff
@@ -212,6 +220,7 @@ func newConfig(buildConfig []byte, rewriteDockerRepositoryName bool) (*Config, e
 		DockerSpec:      dockerSpec,
 		BuilderSpec:     builderSpec,
 		BinaryBuild:     build.Spec.Source.Type == api.BuildSourceBinary,
+		Legacy: legacy,
```

Comment:
Fint om vi kan bruke "BuildahBuild" eller noe. Legacy åpner for tolkning.

Reply:
Fikset